cli.ts

```diff
@@ -6,16 +6,21 @@ import { build } from "./src/build.ts";
 
 await new Command()
   .name("pyro")
-  .version("0.1.0")
+  .version("0.2.1")
   .description("The documentation site generator for Deno")
   .command("gen", "Generate a Pyro site.")
   .arguments("<path:string>")
   .action((_, p) => {
     generate(path.resolve(p));
   })
   .command("dev", "Start the Pyro dev server")
-  .action(() => {
-    dev();
+  .option("-p, --port <value:number>", "Specify a port for the the dev server")
+  .option(
+    "-n, --hostname <value:string>",
```

Comment:
what is the usecase of hostname? I'm a little uninformed on this because I never use it.